.env.compose

```
@@ -557,7 +557,36 @@
+MCP_AUTH_JWT_JWKS_URI=https://auth.gauzy.co/.well-known/jwks.json
+
+# Production authorization server configuration
+MCP_AUTH_SERVERS=[{"issuer":"https://auth.gauzy.co","authorizationEndpoint":"https://auth.gauzy.co/oauth2/authorize","tokenEndpoint":"https://auth.gauzy.co/oauth2/token","registrationEndpoint":"https://auth.gauzy.co/oauth2/register","introspectionEndpoint":"https://auth.gauzy.co/oauth2/introspect","grantTypesSupported":["authorization_code","refresh_token","client_credentials"],"responseTypesSupported":["code"],"scopesSupported":["mcp.read","mcp.write","mcp.admin","openid","profile","email"],"codeChallengeMethodsSupported":["S256"]}]
+
+# Alternative: Token introspection for opaque tokens
+# MCP_AUTH_INTROSPECTION_ENDPOINT=https://auth.gauzy.co/oauth2/introspect
+# MCP_AUTH_INTROSPECTION_CLIENT_ID=gauzy-mcp-server
+# MCP_AUTH_INTROSPECTION_CLIENT_SECRET=secure-client-secret
+
+# Cache settings for performance
+MCP_AUTH_TOKEN_CACHE_TTL="600"     # 10 minutes
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)